app/docs/0.10.x/plugin-development/custom-logic.md

```diff
@@ -85,6 +87,14 @@ function CustomHandler:certificate(config)
   -- Implement any custom logic here
 end
 
+function CustomHandler:rewrite(config)
```

Comment:
Yes there is. The handler is only executed for plugins that have been configured globally. But they still have a config.